apache-example.conf

```diff
@@ -0,0 +1,39 @@
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+ ServerName  my.server.name
+ ServerAlias  my.server.name
+ DocumentRoot /path/to/server-skeleton/
+
+ # Line below assumes that the pytho virtual environment is set up in server-skeleton/.venv
+ WSGIDaemonProcess etes python-path=/path/to/server-skeleton/ python-home=/path/to/server-skeleton/.venv
```

Comment:
I assume etes is the etesync user? Would you mind changing it to EtesyncUser and change the group to EtesyncGroup?